crates/subspace-networking/src/connected_peers.rs

```diff
@@ -435,6 +484,18 @@ impl<Instance: 'static + Send> NetworkBehaviour for Behaviour<Instance> {
             }
         }
 
+        // Log the internal state.
+        match self.logging_delay.poll_unpin(cx) {
+            Poll::Pending => {}
+            Poll::Ready(()) => {
+                self.logging_delay.reset(self.config.logging_interval);
+
+                let stats = self.gather_stats();
+
+                debug!(instance=%self.config.log_target, ?stats, "Connected peers protocol statistics.");
```

Comment:
I'm not *that* opposed to this, but I am also not sure this is the right place to do this. Ideally our farmer would print periodic message like Substrate does with some stats. I understand that this is just for networking, but still seems like we might want to combine everything into one message.